NineChronicles.Headless.Executable/Program.cs

```diff
@@ -307,6 +311,7 @@ static void ConfigureSentryOptions(SentryOptions o)
                             ? new GraphQLNodeServiceProperties.MagicOnionHttpOptions(
                                 $"{headlessConfig.RpcListenHost}:{headlessConfig.RpcListenPort}")
                             : (GraphQLNodeServiceProperties.MagicOnionHttpOptions?)null,
+                        IpRateLimitOptions = rateLimit == true ? configuration.GetSection("IpRateLimiting") : null,
```

Comment:
Because `Configuration.GetSection` returns `null` when the section doesn't exist, who about trying `IpRateLimiting` section without `--rate-limit` flag?

https://learn.microsoft.com/ko-kr/dotnet/api/system.configuration.configuration.getsection?view=dotnet-plat-ext-6.0